app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabViewHolder.kt

```diff
@@ -69,7 +68,10 @@ class RecentTabViewHolder(
             )
 
             recentSyncedTabState.value?.let {
-                if (FeatureFlags.taskContinuityFeature && it != RecentSyncedTabState.None) {
+                if (
+                    components.settings.enableTaskContinuityEnhancements &&
```

Comment:
should we keep the first part of this inline with the `if`?

Reply:
For some reason I'd been under the impression that inlining would cause a lint error, but it apparently does not. I agree it looks better with that suggestion. Updated!